api_tests/lib/cnd/cnd_instance.ts

```diff
@@ -65,19 +65,13 @@ export class CndInstance {
             );
         }
 
-        this.process.on("exit", (code: number, signal: number) => {
```

Comment:
Unfortunately this hook does not work anymore: I get an error because this is printed after the test was executed. Something like: `can't print after tests are done`.

Reply:
Okay, so be it then :)